### DISCUSSION TITLE
Discord: Query regarding usage of '$' symbol

### DISCUSSION THREAD

**Discussion by Gluant101** [gluestack/gluestack-ui#1423] [Q&A]
Walter:

Why do some Gluestack Tokens require the '$' symbol before the token name to work, and why some do not require it and will even throw an error if trying to use the '$' symbol?

for example, when using the Heading component, the color token must use the '$' , but the size must not use the '$'.

This works: 
`<Heading color="$orange600" >My Heading</Heading>
`
This does not works:  
```
<Heading size="$2xl">My Heading</Heading>
 , needs to be 
<Heading size="2xl">My Heading</Heading>
 without the $ symbol
```

This really confuses me 

Jono:
This also the same with useToken. The value has to be $'less to work. Even with color values

**Comment by ankit-tailor** [comment]
Hey @Gluant101, `Token` values are the only ones prefixed with `$`. However, for `variants` like size, which fall under component props, you can directly pass the value without using the `$` prefix.
